### DISCUSSION TITLE
fastring 直接<< u8是空的,转成u16就正常了

### DISCUSSION THREAD

**Discussion by tiantianaixuexi** [idealvin/coost#175] [Q&A] [answered]
```
fastring RedisSql;
uint8_t s = 1;
RedisSql<<"s="<<s;
std::cout<<RedisSql.c_str()<<std::endl;
```
打印结果是乱码

**Comment by idealvin** [comment, marked as answer]
- C++ 中 char, signed char, unsigned char 是 3 种不同的类型，`std::ostream` 对这 3 种类型，都是输出字符。
- int8 有些系统定义为 signed char，uint8 有些系统定义为 unsigned char。
- 若 int8，uint8 输出为整数，可能与 `std::ostream` 行为不一致。
